selectors.yml

```diff
@@ -195,6 +195,9 @@ selectors:
   168587773:
     selector: 2027362563942762617
     name: "ethereum-testnet-sepolia-blast-1"
+  978657:
+    selector: 10443705513486043421
+    name: "ethereum-testnet-sepolia-arbitrum-1-treasure-ruby"
```

Comment:
going by the convention shouldn't this be `ethereum-testnet-ruby-arbitrum-1-treasure-1`?

Reply:
I'm using the same RDD naming: https://github.com/smartcontractkit/reference-data-directory/tree/master/ethereum-testnet-sepolia-arbitrum-1-treasure-ruby/linkTokens

If we rename it here, we might need to rename it for RDD as well.

Reply:
RDD promised to follow the chain-selectors naming convention, we've agreed chain-selectors to be the source of truth

Reply:
@andrevmatos your thought on that?

Reply:
@KodeyThomas Following the convention, it would be then `ethereum-testnet-sepolia-arbitrum-1-treasure-1` right?

Reply:
Great, resolving this by changing the name to: `ethereum-testnet-sepolia-arbitrum-1-treasure-1`